typings/index.d.ts

```diff
@@ -50,8 +50,8 @@ declare module '@dudigital/react-native-zoomable-view' {
     onZoomAfter?: (event: Event, gestureState: PanResponderGestureState, zoomableViewEventObject: ZoomableViewEvent) => boolean;
     onZoomEnd?: (event: Event, gestureState: PanResponderGestureState, zoomableViewEventObject: ZoomableViewEvent) => boolean;
     onLongPress?: (event: Event, gestureState: PanResponderGestureState) => void;
-    onStartShouldSetPanResponder?: (event: Event, gestureState: PanResponderGestureState, zoomableViewEventObject: ZoomableViewEvent, any) => boolean;
-    onMoveShouldSetPanResponder?: (event: Event, gestureState: PanResponderGestureState, zoomableViewEventObject: ZoomableViewEvent, any) => boolean;
+    onStartShouldSetPanResponder?: (event: Event, gestureState: PanResponderGestureState, zoomableViewEventObject: ZoomableViewEvent, baseComponentResult: boolean) => boolean;
```

Comment:
The naming for the parameter is taken from the parent repository docs.

https://github.com/DuDigital/react-native-zoomable-view#pan-responder-hooks